src/jdk.compiler/share/classes/com/sun/tools/javac/util/Filter.java

```diff
@@ -28,7 +28,12 @@
 /**
  * Simple filter acting as a boolean predicate. Method accepts return true if
  * the supplied element matches against the filter.
+ *
+ * @deprecated
+ * It is better to use {@link java.util.function.Predicate}
+ * instead of {@link com.sun.tools.javac.util.Filter}.
  */
+@Deprecated(forRemoval = true, since = "17")
```

Comment:
This is not a public API - we're in a JDK internal class - if this class is no longer needed (as is the goal of this change) simply remove it :-)

Reply:
Fixed.